app/views/queries/index.html.erb

```diff
@@ -13,6 +13,14 @@
       <th>Best bets</th>
       <th>Worst bets</th>
     </tr>
+
+    <tr class="table-header">
```

Comment:
Sorry this feedback comes post-merge.

Typically this feature has the classes `if-no-js-hide table-header-secondary` – which hide the form when there's no javascript and gives the row a slightly lighter background to differentiate it from the header as well as the table content.